pyproject.toml

```diff
@@ -211,9 +211,11 @@ reportUnnecessaryTypeIgnoreComment = true
 reportMissingTypeStubs = false
 exclude = [
     "docs/**/*.py",
+    "examples/**/*.py",
     "site/**/*.py",
     ".venv",
     "venv*",
+    "**/venv*",
```

Comment:
IDK if 217 + 218 are both technically necessary, without 218 pyright was still doing a bunch of type checking on the virtual env specific to my example project.